src/uikit.cs

```diff
@@ -2900,10 +3091,12 @@ interface UIContextMenuInteractionDelegate {
 		[return: NullAllowed]
 		UIContextMenuConfiguration GetConfigurationForMenu (UIContextMenuInteraction interaction, CGPoint location);
 
+		[Deprecated (PlatformName.iOS, 16, 0)]
```

Comment:
Should there be a message for this? I don't know our typical policy for this. Personally, I view deprecation as either a detour sign (what's my alternative?) or an apology.

Reply:
There should, but we do not know the reason, so I can only say it was deprecated by apple.